riptide-spring-boot-autoconfigure/src/main/java/org/zalando/riptide/autoconfigure/DefaultRiptideRegistrar.java

```diff
@@ -147,19 +143,12 @@ private String registerClientHttpRequestFactory(final String id, final Client cl
     private String registerExecutor(final String id, final Client client) {
         // TODO support maxSize = 0 => direct executor?
```

Comment:
Our idea was to go explicit with `threads.enabled: false` for that. Something for a follow-up?